blocks/library/tag-cloud/index.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Server-side rendering of the `core/tag-cloud` block.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Renders the `core/tag-cloud` block on server.
+ *
+ * @param array $attributes The block attributes.
+ *
+ * @return string Returns the tag cloud for selected taxonomy.
+ */
+function gutenberg_render_block_core_tag_cloud( $attributes ) {
+	$align = 'center';
+	if ( isset( $attributes['align'] ) && in_array( $attributes['align'], array( 'left', 'right', 'full' ), true ) ) {
```

Comment:
I wouldn't expect `align` should work here, because you have no attribute schema defined on the server. See the [Latest Posts block implementation](https://github.com/WordPress/gutenberg/blob/b4ba42bee4f767211c1a3b975f940b886c3552f5/blocks/library/latest-posts/index.php#L68-L103) as an example where attributes are to be defined on the server. These will automatically load into the client, so you don't need to duplicate there.

Eventually the idea is that all blocks will be defined this way : #2751